test/test_bulk_actions.py

```diff
@@ -2,6 +2,9 @@
 import pytest
 
 from geant_argus.geant_argus.incidents.bulk_actions import clear_incident_in_metadata
+from datetime import datetime
+from unittest.mock import patch, MagicMock
+from src.geant_argus.geant_argus.incidents.bulk_actions import bulk_clear_incidents
```

Comment:
don't import from `src` here, you should just import from `geant_argus.geant_argus.incidents.bulk_action`. see also line 4 just above